site/content@beta/agent-chat-web-api/methods.md

```diff
@@ -0,0 +1,1604 @@
+| `Content-Type`	 |  `multipart/form-data; boundary=<boundary>`  | Valid for `send_file` and `upload_image` Method URL |
+| `Content-Type` |  `application/json`	    |   Valid for every Method URL except `send_file `and `upload_image` |
+| `Authorization` |  `Bearer <token>`	    |   Access token |
+
+---------------------------------------------------------------
+
+
+|   |  |
+|-------|--------| 
+| **chats** | [`activate_chat`](#activate-chat) [`follow_chat`](#follow-chat) [`get_chats_summary`](#get-chats-summary) [`start_chat`](#start-chat) [`transfer_chat`](#transfer-chat) [`unfollow_chat`](#unfollow-chat) |
+| **chat access** | [`grant_access`](#grant-access) [`revoke_access`](#revoke-access) [`set_access`](#set-access)   |
+| **chat users** | [`add_user_to_chat`](#add-user-to-chat) [`remove_user_from_chat`](#remove-user-from-chat) [`update_agent`](#update-agent)| 
+| **customers** | [`ban_customer`](#ban-customer) [`create_customer`](#create-customer) [`get_customers`](#get-customers) [`set_customers`](#get-customers) [`update_customer`](#update-customer)| 
+| **events** | [`send_event`](#send-event) |
+| **properties (chat/thread/event)** | [`delete_chat_properties`](#delete-chat-properties) [`delete_chat_thread_properties`](#delete-chat-thread-properties) [`delete_event_properties`](#delete-event-properties) [`update_chat_thread_properties`](#update-chat-thread-properties) [`update_chat_properties`](#update-chat-properties) [`update_event_properties`](#update-event-properties) |  
+| **thread tags** | [`tag_chat_thread`](#tag-chat-thread) [`untag_chat_thread`](#untag-chat-thread) | 
+| **other** | [`get_archives`](#get-archives) [`get_chat_threads`](#get-chat-threads) [`send_file`](#send-file) [`get_chat_threads_summary`](#get-chat-threads-summary) [`close_thread`](#close-thread) [`upload_image`](#upload-image) [`multicast`](#multicast) [`update_last_seen_timestamp`](#update-last-seen-timestamp) [`send_typing_indicator`](#send-typing-indicator) [`send_rich_message_postback`](#send-rich-message-postback)| 
+
+## chats
+
+### `activate_chat`
+
+> **`activate_chat`** sample request
+
+   ```json
+  curl "https://api.livechatinc.com/v3.0/agent/action/activate_chat" \
+  -H "Authorization: Bearer <your_access_token>" \
+  -H Content-type:application/json \
+  -H "X-API-Version: 3"
+  ```
+
+```js
+{
+	"chat": {
+		"id": "PJ0MRSHTDG",
+		"access": {
+			"group_ids": [1]
+		},
+		"properties": {
+			"source": {
+				"type": "facebook"
+			}
+		},
+		"thread": {
+			"events": [{
+				"type": "message",
+				"custom_id": "31-0C-1C-07-DB-16",
+				"text": "hello there"
+			}, {
+				"type": "system_message",
+				"custom_id": "31-0C-1C-07-DB-16",
+				"text": "hello there"
+			}],
+			"properties": {
+				"source": {
+					"type": "facebook"
+				},
+				...
+			},
+			"tags": ["bug_report"]
+		}
+	}
+}
+```
+
+#### Specifics
+
+|  |  |
+|-------|--------|
+| **Method URL**   | `https://api.livechatinc.com/v3.0/agent/action/activate_chat`  |
```

Comment:
Zastanawiam się czy warto przy każdej metodzie dodać do tabeli wiersz Headers i wypisać te które są wymagane (powtórzyć to co jest napisane na początku rozdziału Methods) oraz dodatkowe, które występują tylko przy niektórych metodach. Warto, żeby było albo w tej tabelce albo w przykładzie, przy każdej metodzie.